app/services/facilities/website_url_service.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+module Facilities
+  class WebsiteUrlService
+    FACILITY_TYPES = {
+      'va_benefits_facility' => 'VBA',
+      'va_health_facility' => 'VHA',
+      'va_cemetery' => 'NCA',
+      'vet_center' => 'VC'
+    }.freeze
+
+    def initialize
+      source = Rails.root.join('lib', 'facilities', 'website_data', 'websites.csv')
+      station_websites = CSV.read(source, headers: true)
+      @websites = map_websites_to_stations(station_websites)
+    end
+
+    def find_for_station(id, type)
+      unique_id = "#{id}_#{FACILITY_TYPES[type]}"
```

Comment:
```suggestion
      unique_id = "#{id}_#{BaseFacility.PREFIX_MAP[type].upcase}"
```
We could pull the types from `BaseFacility` and eliminate the map at the top of this file.

Reply:
Pulled it from `BaseFacility` 👍